scripts/release-docker.sh

```diff
@@ -188,9 +190,9 @@ if [[ -z "$NOUPLOAD" ]] || [[ "$NOUPLOAD" -eq 0 ]]; then
 
   # push to GCR
   docker push "${TAG}"
```

Comment:
It may help to add a guard around this push statement to verify if the tag already exists in the registry. We can't explicitly guarantee uniqueness in the future with different build types with this solution. It is good for now though. I think it should be possible with `docker manifest inspect`. I wish there was a way to enforce this with the dhall type system, but I'm not sure how.

Reply:
I've added it but docker manifest inspect requires docker cli in experimental flags which is missing on bk agent. Another approach was to use pull but it conflicted with local docker which is always there

Reply:
ok no worries then , if anything that is out of scope for this change given the difficulty. I think this can be a future enhancement as this solves the problem for now.